semantic/src/site/elements/label.overrides

```diff
@@ -22,7 +27,26 @@
   background-color: transparent;
   color: @darkGrey;
   font-weight: @regularWeight;
-  font-size: @fontXXsmall;
   height: 20px;
   padding-left: 0;
 }
+
+.ui.label.text-italic {
+  font-style: italic;
+}
+
+.ui.label.simple-label > .text-blue {
+  color: @information;
+}
+
+.ui.label.link-label {
+  background-color: transparent;
+  color: @blue;
+}
+
+.ui.active.label.link-label:hover {
+  background-color: transparent;
+  border-color: none;
+  opacity: @hoverOpacity;
+  color: @blue;
+}
```

Comment:
You should be able to achieve this by just over-riding the variables, I don't think you need to override the class definitions. (since it's really just colours your re-mapping, right?)

Reply:
For example, see "Form.variables" -- there's a bunch of variables where I've just over-ridden the colour definitons to point to our colour palette.

Reply:
Sorry, end up not using this, using the `<Icon link ...>` styled seemed as a better approach. Will remove this unused definitions now.